launch/tier4_perception_launch/launch/object_recognition/prediction/prediction.launch.xml

```diff
@@ -2,11 +2,13 @@
 <launch>
   <arg name="use_vector_map" default="false" description="use vector map in prediction"/>
   <arg name="param_path" default="$(find-pkg-share map_based_prediction)/config/map_based_prediction.param.yaml"/>
+  <arg name="input_topic_to_prediction" default="/perception/object_recognition/tracking/objects"/>
```

Comment:
[IMO]
I think `input_topic_to_prediction` should change into `input/objects` because this file is prediction.launch.xml.
How about unifying the input value as below perception.launch?

```
     <group>
        <push-ros-namespace namespace="detection"/>
        <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/detection/detection.launch.xml">
          <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
        </include>
      </group>

      <!-- Tracking module -->
      <group>
        <push-ros-namespace namespace="tracking"/>
        <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/tracking/tracking.launch.xml">
          <arg name="input/objects" value="detection/objects"/>
        </include>
      </group>

      <!-- Prediction module -->
      <group>
        <push-ros-namespace namespace="prediction"/>
        <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/prediction/prediction.launch.xml">
          <arg name="input/objects" value="tracking/objects"/>
        </include>
      </group>
    </group>
```

Reply:
I just refactored `prediction.launch.xml` part in https://github.com/autowarefoundation/autoware.universe/pull/5269/commits/c1e95dc8d76dd89bc0b0f76142e3b557a03a7aee.